Makefile

```diff
@@ -277,22 +278,22 @@ update-downloadcache: $(CACHE)
 # DEPS
 ######
 # Use the pyramid install dir as our indicator that dependencies are installed.
-$(PYRAMID): $(PY) $(CACHE)
+$(PYRAMID): $(PY) $(CACHE) requirements.txt
 	$(call PIP,requirements.txt)
+	@touch $(PYRAMID)
```

Comment:
Is this canary necessary? In my (limited) experience with pip it appears that it only installs items which aren't installed and skips ones which are already installed.